src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
```

Comment:
It's not recommended to use tag names for styling (except `html` and `body`). However, in this case, it's acceptable because you're setting a global style. Still, be mindful of this rule for other elements.